tests/config/full_params_2022Q4.json

```
@@ -0,0 +1,76 @@
+{
+  "holdingsDate": "2022Q4",
+  "pactaDataURL": "https://pactadatadev.blob.core.windows.net/pacta-data-webapp/2022Q4/2022Q4_20240426T113151Z",
+  "benchmarksURL": "https://pactadatadev.blob.core.windows.net/benchmarks-webapp/2022Q4/2022Q4_20240529T002407Z",
+  "parameters": {
+    "portfolio": {
+      "holdingsDate": "2023-12-31",
```

```suggestion
      "holdingsDate": "2022-12-31",
```

Relates to https://github.com/RMI-PACTA/workflow.pacta/pull/105